doc/manuals/admin/perf_tuning.md

```diff
@@ -122,6 +123,60 @@ on [the `notifQueue` block](statistics.md#notifqueue-block) may help you to tune
 
 [Top](#top)
 
+## Updates flow control mechanism
+
+Orion implements a flow control mechanism that allow to slow-down the updates flow sent by a client.
+Using flow control, Orion does not respond immediately to update request, but when the notifications
+triggered by the update has been sent (totally or partially, depending on the flow control
+configuration). This way, we avoid Orion saturation due to too much accumulated notifications
+in the notification queue (which, at the end, causes notifications being discarded when the queue
+is full).
+
+Flow control mechanism is configured using `-notifFlowControl` which takes three parameters:
+`-notifFlowControl gauge:stepDelay:maxInterval`. It requires threadpool notification mode
```

Comment:
Three parameters. It is not clear the meaning of each parameter and its possible values.

Reply:
Actually is explained below. However, at this point the reader don't know it. I have slightly modified the text in 9a8663688bda2800ec8dc92b917cc2cb4010f447 to make it clearer.